docs/reference/api.rst

```diff
@@ -20,6 +20,14 @@ Constants
     :members:
     :undoc-members:
 
+.. only:: iso
```

Comment:
I tested this by changing it to `.. only:: oss ` to see if it was displayed when running `make docs html` locally (and it was). and then changing it to `.. only:: iso` (and then it was not). I don't know how to build the iso docs locally and I didn't find a jenkins job to which I could give my branch as parameter. I also didn't find anything about this in the internal docs.